test/lit/exec/nontrapping-fptoint-lowering.wat

```diff
@@ -0,0 +1,231 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
+
+;; RUN: wasm-opt --enable-nontrapping-float-to-int %s --llvm-nontrapping-fptoint-lowering -q -o /dev/null 2>&1 | filecheck %s
+
+(module
+
+  (func $assert_i32 (param i32 i32)
+    (if (i32.ne (local.get 1) (local.get 0))
+    (then (unreachable)))
+  )
+  (func $assert_i64 (param i64 i64)
+    (if (i64.ne (local.get 1) (local.get 0))
+    (then (unreachable)))
+  )
+
+  (func $i32.trunc_sat_f32_s (param $x f32) (result i32) (i32.trunc_sat_f32_s (local.get $x)))
+  (func $i32.trunc_sat_f32_u (param $x f32) (result i32) (i32.trunc_sat_f32_u (local.get $x)))
+  (func $i32.trunc_sat_f64_s (param $x f64) (result i32) (i32.trunc_sat_f64_s (local.get $x)))
+  (func $i32.trunc_sat_f64_u (param $x f64) (result i32) (i32.trunc_sat_f64_u (local.get $x)))
+  (func $i64.trunc_sat_f32_s (param $x f32) (result i64) (i64.trunc_sat_f32_s (local.get $x)))
+  (func $i64.trunc_sat_f32_u (param $x f32) (result i64) (i64.trunc_sat_f32_u (local.get $x)))
+  (func $i64.trunc_sat_f64_s (param $x f64) (result i64) (i64.trunc_sat_f64_s (local.get $x)))
+  (func $i64.trunc_sat_f64_u (param $x f64) (result i64) (i64.trunc_sat_f64_u (local.get $x)))
+
+  (func $f32_i32 (export "f32_i32")
```

Comment:
The CHECK lines don't seem to be here? May need to run `./scripts/update_lit_checks.py test/lit/exec/*`

Reply:
ah oops, yes, the fuzz comparison runs, but the test run was written not to generate output. I think I pasted that from somewhere, I guess some tests want to check the output and some don't.

Reply:
Actually, does it make more sense just to not print the wast output in this test? test/lit/passes/nontrapping-fptoint-lowering.wast prints the output, and the output from this test is the same.

Reply:
The wast isn't really needed, I agree. But we should still have the execution output, even if just to report that the function was called and returned nothing.

Actually, I see `--fuzz-exec` is missing from line 3, is that the issue maybe?

Reply:
Yes, that was why the tests didn't get auto-updated. Fixed now, using just the fuzz-exec output.